app/components/results-page.js

```diff
@@ -1,4 +1,33 @@
 import Component from "@ember/component";
+import { computed } from "@ember/object";
 
 export default Component.extend({
+  score: null,
+  stepsForward: computed("answers", function() {
+    const numSteps = this.answers.filter(a => a === 1).length;
```

Comment:
This is using the answers array from state instead of from the DB. Since we store the answers array from state directly into the DB, I don't see why the two would ever get out of sync. As long as that's true I _think_ using the state here is okay.